homeassistant/components/wallbox/sensor.py

```diff
@@ -138,7 +147,10 @@ async def async_setup_entry(
 ) -> None:
     """Create wallbox sensor entities in HASS."""
     coordinator: WallboxCoordinator = hass.data[DOMAIN][entry.entry_id]
-
+    is_quasar = coordinator.data[CHARGER_DATA_KEY][CHARGER_PART_NUMBER_KEY].startswith("QSX")
+    
+    if not is_quasar: 
+        del SENSOR_TYPES[CHARGER_ADDED_DISCHARGED_ENERGY_KEY]
```

Comment:
I would just add the sensor without the delete part, I have a lot of sensors in hass that I don't use, one more won't make a difference ;-)